runtime/near-vm-logic/src/logic.rs

```diff
@@ -24,6 +25,51 @@ pub type Result<T> = ::std::result::Result<T, VMLogicError>;
 const LEGACY_DEFAULT_PROTOCOL_VERSION: ProtocolVersion = 34;
 const IMPLICIT_ACCOUNT_CREATION_PROTOCOL_VERSION: ProtocolVersion = 35;
 
+pub fn transfer_exec_fee(
```

Comment:
why does this function need to be public? Also it probably makes more sense to implement this in `runtime/src/config.rs`

Reply:
Moved to `primitives-core/src/runtime/fees.rs`, because the `runtime` package depends on `near-vm-logic`.